orderfile/app/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.18.1)
+project(OrderfileDemo CXX)
+
+set(GENERATE_PROFILES ON)
+#set(USE_PROFILE "${CMAKE_SOURCE_DIR}/demo.orderfile")
```

Comment:
Best to add a comment here explaining how to use this. Docs are more useful if they're right next to the relevant code :) (keep the thing in the readme as a signpost so people know how to find this though)